code_challenges.rb

```diff
@@ -33,19 +47,98 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+def vowel_delete string
+    string.downcase.delete('aeiou')
+end
+
+p vowel_delete(album1)
 
+p vowel_delete(album2)
 
+p vowel_delete(album3)
 
 
 # --------------------4) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a get_info method that returns a sentence with all the data from the bike object.
 
 # Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
 
+class Bike 
+    attr_accessor :model, :wheels, :current_speed, :speed
+
+    def initialize (model, wheels, current_speed)
+        @model = model
+        @wheels = 2
+        @current_speed = 0
+    end
 
+    def speed_bike
```

Comment:
consider providing the method a parameter for the amount the speed increases instead of hardcoding it to 1